Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -461,9 +507,12 @@ def publish_bottle_file_on_bintray(f, bintray_org, creds)
          "https://api.bintray.com/content/#{bintray_org}/#{repo}/#{package}/#{version}/publish"
     true
   rescue => e
-    raise unless @args.warn_on_publish_failure?
-
-    onoe e
+    if @args.warn_on_publish_failure?
+      ohai "No bottle found on Bintray for: #{package} #{version} - skipping"
```

Comment:
How is this change related?

Reply:
It's just cosmetics. But I am still unhappy with the result. I will go back to the initial code, this change is unrelated. I'll open a separate PR for this later.